othdrplot/process_high_density_region_algorithm.py

```diff
@@ -127,67 +127,32 @@ def drawDimensionReduction(self):
         return graph
 
     def drawDensity(self, drawData=False, drawOutliers=True):
-        """Density draw based on HDR.
+        """Draw contour.
 
         If :attr:`drawData`, the whole sample is drawn. Otherwise, depending on
         :attr:`drawOutliers` it will either show the outliers or the inliers
         only.
 
         :param bool drawData: Plot inliers and outliers.
         :param bool drawOutliers: Whether to draw inliers or outliers.
-        :return: HDR in an OpenTURNS graph object.
-        :rtype: :class:`openturns.Graph`
+        :returns: figure, axes and OpenTURNS Graph object.
+        :rtypes: Matplotlib figure instances, Matplotlib AxesSubplot instances,
+          :class:`openturns.Graph`
         """
-        graph = ot.Graph('High Density Region draw', '', '', True, 'topright')
-
-        graph.add(self.densityPlot.drawContour())
-
-        if drawData:
-            graph.add(self.densityPlot.drawInliers())
-            graph.add(self.densityPlot.drawOutliers())
-        elif drawOutliers:
-            graph.add(self.densityPlot.drawOutliers())
-        else:
-            graph.add(self.densityPlot.drawInliers())
-
-        return graph
-
-    def drawTrajectories(self, discreteMean=False):
-        """Plot trajectories from the :attr:`ProcessSample`.
+        return self.densityPlot.drawContour(drawData=False, drawOutliers=True)
```

Comment:
This line should be replaced with:

return self.densityPlot.drawContour(drawData, drawOutliers)

because otherwise it does not take into account the drawData and drawOutliers arguments of the function.